### DISCUSSION TITLE
Should template styles be in fast-foundation?

### DISCUSSION THREAD

**Discussion by bartcorremans** [Adaptive-Web-Community/Adaptive-Web-Components#51] [Ideas]
Hi! I've been working on a component library/design system extending FAST foundation for a while now, and I'm excited to see these new developments of the adaptive capabilities.

It's great to see the separation of template and aesthetic styles in `adaptive-web-components`, which should allow for leaner CSS in custom design systems, focused only on the parts that they're actually interested in customising.

I'm curious what the motivation is to not include these core styles in `fast-foundation`. As I understand it, template styles generally contain the CSS that is required for the component's template to be rendered as intended. Anyone extending a foundation component would inevitably end up implementing some basic CSS that is at least very similar to these template styles.

If it is a concern that even the template styles would be too opinionated for FAST, would this not be addressed by having them be optional imports?

I'm interested to hear your thoughts on this.

**Comment by KingOfTac** [comment]
I think this would be a good question to ask in the FAST discord. This project, while built by some of the FAST team isn't a FAST project or really affiliated with it other than consuming the FAST packages.
